torch/_inductor/kernel/mm.py

```diff
@@ -183,6 +189,11 @@ def tuned_mm(mat1, mat2, *, layout=None):
         log.warning("No choices for GEMM, using ATen backend as fallback")
         return aten_mm.bind((mat1, mat2), aten_layout).output_node()
 
+    from ..config import external_matmul
+
+    for k in external_matmul:
```

Comment:
```suggestion
    for k in config.external_matmul:
```

config is backed by a dictionary, so you should use it this way

Reply:
Fixed